web/site.py

```diff
@@ -58,6 +58,45 @@ def search():
             else:
                 error = "Ungueltiges Bild"
 
+    if "nsfl" not in request.form.getlist("filter"):
+            while True:
+                nsfl_c = 0
+                for u in search_results:
+                    if u.post.flags == 4:

```

Comment:
`Post` hat ein paar Methoden mit denen überprüft werden kann, in welche Kategorie das Bild gehört: https://github.com/ReneHollander/rep0st/blob/master/rep0st/database.py#L64